editor/plugins/script_text_editor.cpp

```diff
@@ -1392,16 +1392,14 @@ void ScriptTextEditor::_change_syntax_highlighter(int p_idx) {
 
 void ScriptTextEditor::_notification(int p_what) {
 	switch (p_what) {
-		case NOTIFICATION_THEME_CHANGED:
+		case NOTIFICATION_THEME_CHANGED: {
 			if (!editor_enabled) {
 				break;
 			}
 			if (is_visible_in_tree()) {
 				_update_warnings();
 				_update_errors();
 			}
-			[[fallthrough]];
-		case NOTIFICATION_ENTER_TREE: {
```

Comment:
Worth noting that it's not exactly equivalent (now it will reset the gutter width on all theme changes), but it's probably fine anyway.

Reply:
Fixed